<?xml version="1.0"?>
<launch>


	<!-- Nodes to create kalman filters -->
	<group ns = "uav0">
        <include file="$(find sensor_fusion)/launch/uav_sitl.launch" >
            <arg name="ID" value="0"/>
        </include>
    </group>

    <group ns = "uav1">
        <include file="$(find sensor_fusion)/launch/uav_sitl.launch" >
            <arg name="ID" value="1"/>
        </include>
    </group>

    <group ns = "uav2">
        <include file="$(find sensor_fusion)/launch/uav_sitl.launch" >
            <arg name="ID" value="2"/>
        </include>
    </group>

    <node pkg="offboard_py" type="image_car_detector_multi.py" name="detector_multi" required="True" output="screen">
	</node>




    <include file="$(find offboard_py)/launch/sensor_camera.launch">
		<arg name = "uav_id" value = "uav0"/>
	</include>

	<include file="$(find offboard_py)/launch/sensor_camera.launch">
		<arg name = "uav_id" value = "uav1"/>
	</include>
    
    <include file="$(find offboard_py)/launch/sensor_camera.launch">
		<arg name = "uav_id" value = "uav2"/>
	</include>








</launch>
